### ISSUE TITLE
Commands sent from toolbar are sent twice

### ISSUE THREAD

**Issue by demitrix** [jimradford/superputty#732]
**Describe the bug**
Typing a command in the toolbar and using the send command or enter results in some sessions receiving the command twice.

**To Reproduce**
1. Open multiple sessions
2. Enter command in toolbar and send with button or enter
3. Command is executed twice on some session, once on others.

**Expected behavior**
Command to be sent only once to all sessions

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version 1.4.0.9

**Comment by RacoonSN** [comment]
I confirm this behavior.
Version 1.4.09
Windows 7

**Comment by demitrix** [comment]
Not a solution to the cause of the bug a resolution for when it does happen. If the session with the error is closed then readded to the layout it will stop receiving commands twice.

**Comment by rste79** [comment]
I have the same issue (1.4.09 - windows 10) and I noticed another issue which I think is related to this one:

![immagine](https://user-images.githubusercontent.com/1794174/58421513-6898c880-8090-11e9-9d20-f0e3b67f218a.png)

As you can see from the screenshot above the tab named "3" is present twice in the "Select document" window, but I can only see one. In SuperPutty window I have only 4 terminals, not 5.

**Comment by md3359** [comment]
I can confirm 1.4.0.9 - Windows 10

**Comment by jkougoulos** [comment]
I see the same in 1.4.0.9 in Windows 10 and Windows 7.
For me it happens even in one session, if the session opens or is moved to a floating window. 
I think that  "Restrict Floating Windows" option might be used as a workaround

**Comment by manmadehuman** [comment]
Same happens to me.  Win10 and v1.4.0.9

I tried using "Restrict Floating Windows" as a workaround but did not change the behavior.  Currently have one particular connection that is doing this consistently.  

I'm going to try an older version and see if it does the same thing.

**Comment by manmadehuman** [comment]
I've been updating switches all day and v1.4.0.8 has not once duplicated a command. I'm suspecting this bug is only in 1.4.0.9

**Comment by nitramartinn** [comment]
The issue can be reproduced on both 1.4.0.8 with Win10 and 1.4.0.9 with Win10. To trigger the issue, just move a session window as the floating window. Then, the command to that session will be sent twice. Restarting the SuperPuTTY will still keep the "send twice" effect. To stop "send twice" effect, the session has to be closed and re-opened. Or, reload the layout may stop the effect. 

The "send twice" session will show duplicated entries on "select document".

**Comment by moraldilema** [comment]
Still seeing this behavior in 1.4.0.9.  Was there ever a fix/workaround for this multiple send from command bar?

**Comment by JH0147** [comment]
Still seeing the same issue in 1.5.0 using windows 11 22H2

**Comment by jkhh07** [comment]
Still seeing this behavior in Windows 11 22H and I may know how this happens.

1. Open session to host_A

3. Open saved layout that includes host_A 
 
5. Open document list as shown in post by @rste79 ^ I have the same issue (1.4.09 - windows 10) and I noticed another issue which I think is related to this one:
> 
> ![immagine](https://user-images.githubusercontent.com/1794174/58421513-6898c880-8090-11e9-9d20-f0e3b67f218a.png)
> 
> As you can see from the screenshot above the tab named "3" is present twice in the "Select document" window, but I can only see one. In SuperPutty window I have only 4 terminals, not 5.
